dcap/sys/types/build.rs

```diff
@@ -40,12 +40,6 @@ const DCAP_TYPES: &[&str] = &[
 fn main() {
     let callback = SgxParseCallbacks::new(
```

Comment:
Can't change this in a GH comment, but having this be `default().enum_types([ ... ])` would be the goal

Reply:
done